spec/features/auth/provider_user_signs_in_spec.rb

```diff
@@ -12,6 +12,20 @@
     and_i_cannot_access_the_support_interface
   end
 
+  context "publish migration" do
+    before do
+      allow_any_instance_of(ActionDispatch::Request).to receive(:referer).and_return(Settings.publish_url)
+      disable_features(:send_request_data_to_bigquery)
+      enable_features(:display_migration_signin)
+    end
+
+    scenario "users are shown the right signin content" do
```

Comment:
In relation to https://github.com/DFE-Digital/teacher-training-api/pull/2411/files#r782211217 :
```suggestion
    scenario "users are shown the right signin content when arriving from Old Publish" do
```